Server/mods/deathmatch/logic/CMainConfig.cpp

```diff
@@ -336,6 +336,21 @@ bool CMainConfig::Load()
         g_pNetServer->SetChecks(strDisableComboACMap, strDisableACMap, strEnableSDMap, m_iEnableClientChecks, iHideAC != 0, strImgMods);
     }
 
+    {
+        SString strDebugLevel;
+        GetString(m_pRootNode, "scriptdebugloglevel", strDebugLevel);
+
+        std::vector<SString> debugLevelList;
+        ReadCommaSeparatedList(strDebugLevel, debugLevelList);
+        for (const auto& level : debugLevelList)
+        {
+            if (!isdigit(**level))
```

Comment:
Maybe better to use `strtol` and the error checking?

Reply:
Actually, wouldnt `isdigit` be more than perfect here? Max value is 4, if it goes above that it should ignore it.
Conversions would be a little expensive here (try-catch, performance costs (not that they matter a lot))

Reply:
Alright